bibliopixel/drivers/hue.py

```diff
@@ -62,7 +61,7 @@ def _rgb2hs(self, rgb):
     def _send_packet(self):
         for i in range(len(self._ids)):
             h, s = self._rgb2hs(self._colors[i + self._pos])
-            bri = self._brightness
+            bri = min(254, self._brightness)
```

Comment:
That comes from you, not me.  :-D

The hue seems to have a maximum brightness limit of 254, which you ensure in one codepath but not another.